app/views/pages/index.html.haml

```diff
@@ -1,2 +1,40 @@
-%h1 Pages#index
-%p Find me in app/views/pages/index.html.haml
+/ Sale and testimonials
+.relative.overflow-hidden.-mx-10.-mt-6
+  / Decorative background image and gradient
+  .absolute.inset-0{"aria-hidden" => "true"}
+    .absolute.inset-0.max-w-7xl.mx-auto.overflow-hidden.xl:px-8
+      -# %img.w-full.h-full.object-center.object-cover{alt: "", src: "https://tailwindui.com/img/ecommerce-images/home-page-02-sale-full-width.jpg"}/
+      .w-full.h-full.object-center.object-cover= link_to image_tag(asset_path('cube.jpg'), alt: "#{(t :sitename)} Logo, link to home page.", id: root_path(anchor: 'page-bottom')), root_path
```

Comment:
@dschmura  I am not finding the cube jpg in the project anywhere. Did you intend on adding this or was this being delivered from Font-awesome?